components/base/edit/section/delete.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <DsfrModal
+    :actions="actions"
+    :opened="true"
+    :title="modalTitle"
+    @close="onClose()"
```

Comment:
```suggestion
    title=""Supprimer la rubrique à la une""
    @close="$emit('close')"
```
me parait un peu plus simple mais chang pas gd chose

Reply:
ca me permet de faire ça plus bas : `onClick: onClose`  et d'être plus verbeux dans les actions